cheatengine.txt

```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = Cheat Engine
+Version = 6.2
+License = Freeware
+Description = Cheat Engine is an open-source tool for modifying game variables by scanning and altering memory addresses.

```

Comment:
You say in description that it's open source, but license is incorrectly set to "Freeware".

Reply:
I looked it up right now and apparently it is "source available" but no specific license

Reply:
well they do accept pull requests so i guess we can call it open source?
https://github.com/cheat-engine/cheat-engine

Reply:
https://github.com/cheat-engine/cheat-engine/issues/60#issuecomment-467179943